templates/procuration/election_rounds.html.twig

```diff
@@ -24,42 +40,24 @@
-                <li><a href="{{ path('app_procuration_request_profile') }}">Mes coordonnées</a></li>
+                <li><a href="{{ path('app_procuration_request', {step: constant('STEP_URI_VOTE', procuration_form.vars.data)}) }}">Mon lieu de vote</a></li>
+                <li><a href="{{ path('app_procuration_request', {step: constant('STEP_URI_PROFILE', procuration_form.vars.data)}) }}">Mes coordonnées</a></li>
                 <li class="active">Ma procuration</li>
             </ul>
             <form method="post" name="app_procuration_elections">
-                {{ form_errors(form) }}
+                {{ form_errors(procuration_form) }}
 
                 <h5 class="procuration__title-elections">
-                    Je donne procuration ...
+                    Je donne procuration pour...
                     <span class="text--small text--gray">(cocher les cases correspondantes)</span>
                 </h5>
 
-                <div class="procuration__choose-election">
-                    <div class="form__checkbox">
-                        {{ form_widget(form.electionLegislativeFirstRound) }}
-                        {{ form_label(form.electionLegislativeFirstRound, 'Pour le premier tour des élections législatives') }}
-                    </div>
-                    <div class="text--small text--body procuration__choose-election__day">
-                        Dimanche 11 juin 2017 en France (3 juin pour les Français de l'étranger du continent Américain et 4 juin pour les autres Français de l'étranger)
-                    </div>
-                </div>
-
-                <div class="procuration__choose-election">
-                    <div class="form__checkbox">
-                        {{ form_widget(form.electionLegislativeSecondRound) }}
-                        {{ form_label(form.electionLegislativeSecondRound, 'Pour le second tour des élections législatives') }}
-                    </div>
-                    <div class="text--small text--body procuration__choose-election__day">
-                        Dimanche 18 juin 2017 en France (17 juin pour les Français de l'étranger du continent Américain et 18 juin pour les autres Français de l'étranger)
-                    </div>
-                </div>
+                {{ form_widget(procuration_form.election_rounds) }}
 
                 <div class="form__row b__nudge--top-40">
-                    {{ form_label(form.reason, 'Pour quelle raison vous est-il impossible de voter ?') }}
-                    {{ form_errors(form.reason) }}
-                    {{ form_widget(form.reason, { 'attr': { 'class': 'form--full b__nudge--top-5' } }) }}
+                    {{ form_label(procuration_form.reason, 'Pour quelle raison vous est-il impossible de voter ?') }}
+                    {{ form_errors(procuration_form.reason) }}
+                    {{ form_widget(procuration_form.reason, { 'attr': { 'class': 'form--full b__nudge--top-5' } }) }}
```

Comment:
In general, can you remove the useless simples quotes for array keys ?

Reply:
Please keep CS comments for final reviews, I'm still in WIP and this is not my code.

Reply:
Now done.